Week-1/index.html

```diff
@@ -0,0 +1,182 @@
+        } */
+        .catty{
+            color:blue;
+            border: 1px solid black;
+        }
+        .icons{
+            display: flex;
+            flex-direction: row;
+        }
+        .merge{
+            display: flex;
+            flex-direction: row;
+        }
+        .icon{
+            padding-right: 80px;
+        }
+        .songoptions{
+            display: flex;
+            flex-direction: row;
+        }
+        .songoption{
+            padding-left: 20px;
+        }
+    </style>
+</head>
+<body>
+    <div class="outer">
+        <div class="sidenav">
+           <div ><img src="./icons8-menu-30.png" alt=""></div>
+           <div class="item"><img src="./icons8-home-32.png" alt="Home"></div>
+           <div class="item"><img src="./icons8-search-30.png" alt="Search"></div>
+           <div class="item"><img src="./icons8-speaker-24.png" alt=""></div>
+           <div class="item"><img src="./icons8-user-30.png" alt=""></div>
+           <div class="item"><img src="./icons8-automation-30.png" alt=""></div>
+           <div class="item"><img src="./icons8-spotify-30.png" alt=""></div>
+           <div class="item"><img src="./icons8-soundcloud-30.png" alt=""></div>
+        </div>
+    <div class="main">
+    <div class="row">
+        <img class='image' src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg" alt="CattyBoard">
+        <div class="content">
+        <h1 class="heading">CattyBoard Top 100 Single Charts (11.06.36)</h1>
+        <p>Unknown Artist</p>
+        <p>2016.Charts.100songs</p>
+        <div class="songoptions">
+            <img src="./icons8-play-30.png" alt="">
+            <p class="songoption">Play All</p>
+            <img  class="songoption"src="./icons8-plus-30.png" alt="">
+            <p class="songoption">Add To</p>
+            <img  class="songoption"src="./icons8-more-30.png" alt="">
+            <p class="songoption">More</p>
+        </div>
+        </div>
+    </div>
+    <table class = 'songs'>
+        <tr>
+            <td>1. One Dance</td>
+            <td>Crake feat CatKid & Cyla</td>
+            <td>2:54</td>
+            <td class="design"><span class="catty"> CATTY CLOUD SYNC</span></td>
+        </tr>
+        <tr>
+            <td>2. Panda</td>
+            <td>Cattee</td>
+            <td>4:06</td>
+            <td class="design"><span class="catty">CATTY CLOUD SYNC</span></td>
+        </tr>
+        <tr>
+            <td>3. Can't Stop the Feeling</td>
+            <td>Catin Cimberlake</td>
+            <td>3:56</td>
+            <td class="design"><span class="catty">CATTY CLOUD SYNC</span></td>
+        </tr>
+        <tr>
+            <td>4. Work From Home</td>
+            <td>Cat Harmony feat Colla</td>
+            <td>3:34</td>
+            <td class="design"><span class="catty">CATTY CLOUD SYNC</span></td>
+        </tr>
+    </table>
+    </div>
+    </div>
+    <div class="foot">
```

Comment:
@relhanakshay can you change this to use semantic tags (header, footer, section, aside etc)
FYI - https://developer.mozilla.org/en-US/docs/Glossary/Semantics